imports/plugins/included/connectors-shopify/lib/collections/schemas/shopifyConnect.js

```diff
@@ -1,6 +1,37 @@
 import { SimpleSchema } from "meteor/aldeed:simple-schema";
 import { PackageConfig } from "/lib/collections/schemas/registry";
 
+const Webhook = new SimpleSchema({
+  shopifyId: {
+    type: Number,
+    label: "Shopify Webhook ID",
```

Comment:
Let's get our casing and language consistent here.

Reply:
Sentence case for labels, correct?